lib/browser_crawler.rb

```diff
@@ -1,3 +1,4 @@
+require 'csv'
```

Comment:
In my opinion requiring `csv` is not needed here, we should require the library only where we need it.
- this is a general namespace which loads itself
- e.g.: `csv` library would not be needed if this will be called with `yaml` report

Reply:
I agree with you.